docker/test-server/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM python:3.9.0-alpine
+
+ARG ETESYNC_VERSION
+
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
+ENV PIP_NO_CACHE_DIR=1
+
+# install packages and pip requirements first, in a single step,
+COPY /requirements.txt /requirements.txt
+RUN set -ex ;\
```

Comment:
You probably want to move the `apk add` to its own `RUN` above the requirements, so you never need to reinstall all of these deps.

Reply:
The apk install is quite quick, and I did this because lots of that stuff gets uninstalled later in the same step.  Breaking it over two steps means that there's a docker layer containing those unused, deleted packages, increasing the image size.  It's the `pip install` that takes a long time.